src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileMetadataRouteSpec.scala

```diff
@@ -466,6 +466,38 @@ class FileMetadataRouteSpec extends TestContainerUtils with Matchers with TestRe
     checkNoFileMetadataAdded(utils, "property2")
```

Comment:
The other tests will need to be re-worded to make it explicit that the "Not Authorised" exception is thrown when the user doesn't own the file and the backend checks client is not used.